server/src/services/checkSubmission.js

```diff
@@ -0,0 +1,174 @@
+class checkSubmission {
```

Comment:
capitalize the class! 

Going into postGrad, I want you hyper on top of variable names, indentation, pluralization and capitalization :P Super important!